tests/std/tests/Dev11_0000000_user_defined_literals/test.cpp

```diff
@@ -560,9 +560,9 @@ int main() {
         assert(str1 == "ct flffy kttns");
         assert(str1_removed == 5);
 
-        std::string str2{"visual studio"};
-        const auto str2_removed = std::erase(str2, 'u');
-        assert(str2 == "visal stdio");
+        std::string str2{"broad anticaustic"};
+        const auto str2_removed = std::erase(str2, 'o');
+        assert(str2 == "brad anticaustic");

```

Comment:
```suggestion
        std::string str2{"asynchronous beat"};
        const auto str2_removed = std::erase(str2, 'a');
        assert(str2 == "synchronous bet");
```

Reply:
Oh, oops. That was my fault, I did a bad job copy+pasting. It was supposed to be identical. I really did know what I was trying to do.

But even so, I like your word pair much better.